frontend/Interfaces/lib/CMakeLists.txt

```diff
@@ -1,39 +1,43 @@
 if (${BUDDY_DIP_OPT_STRIP_MINING})
-  set(SPLITING_SIZE ${BUDDY_DIP_OPT_STRIP_MINING})
-elseif(HAVE_AVX512)
-  set(SPLITING_SIZE 256)
-elseif(HAVE_AVX2)
-  set(SPLITING_SIZE 128)
-elseif(HAVE_SSE)
-  set(SPLITING_SIZE 64)
-elseif(HAVE_NEON)
-  set(SPLITING_SIZE 64)
-endif()
+    set(SPLITING_SIZE ${BUDDY_DIP_OPT_STRIP_MINING})
+elseif (HAVE_AVX512)
+    # TODO: current rotation generates wrong image using avx512
```

Comment:
What do you mean by wrong? Can you please elaborate? Also, please link corresponding github issue here.

Reply:
The output of the picture is wrong. It only happens in new code. I'll open an issue to solve this problem.

Reply:
Does it work properly on other backends? (AVX2, NEON)

Reply:
AVX2 works properly. I don't have NEON machine.

Reply:
This is strange behaviour, I would prefer accuracy over most things. Did you investigate the reason behind wrong image output on AVX 512?

Reply:
Bug fixed.